Hansol_kim/db/migrations/20221107051118_create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+-- migrate:up
+CREATE TABLE users(
+id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+name VARCHAR(50) NOT NULL,
+email VARCHAR(200) NOT NULL,
+profile_image VARCHAR(1000) NULL,
+password VARCHAR(200) NOT NULL,
+image_url VARCHAR(2000) NULL,
+created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP
+);
```

Comment:
```suggestion
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP
    CONSTRAINT users_email_unq UNIQUE (email)
);
```
백엔드 API 서버의 Database에 동일한 회원이 중복으로 가입할 수 없게 email column을 통해 제약 조건을 추가해주세요-!